templates/tsConstructs/constructor.ejs

```diff
@@ -4,11 +4,11 @@
 if(signatures && signatures[0]){
     if (signatures[0].type.type === 'reference') {
     %>
-    <h4 class="code-ref"><%-name%> (<%- include('commaSeperatedParams', {'params': signatures[0].parameters}); %>): <a href="#<%-signatures[0].type.name%>"><%-signatures[0].type.name%></a></h4>
+    <h4 class="code-ref"><%-name%>( <%-include('commaSeperatedParams', {'params': signatures[0].parameters});%> ) : <a href="#<%-signatures[0].type.name%>"><%-signatures[0].type.name%></a></h4>
     <%
     } else {
     %>
-    <h4 class="code-ref"><%-name%> (<%- include('commaSeperatedParams', {'params': signatures[0].parameters}); %>): <%-signatures[0].type.name%></h4>  
+    <h4 class="code-ref"><%-name%>( <%-include('commaSeperatedParams', {'params': signatures[0].parameters});%> ) : <%-signatures[0].type.name%></h4>  
```

Comment:
No this changes "Ctor ( a, b, c )" to "Ctor( a, b, c )".   (took out space between "Ctor" and "(").
The space between "(" and "a", and "c" and ")" still needs to be taken out, but thats out of my control. EJS templating is doing that.